controllers/admin/AdminStoresController.php

```diff
@@ -566,6 +566,17 @@ public function beforeUpdateOptions()
         }
     }
 
+    public function updateOptionPsShopName($value)
+    {
+        if (!Shop::isFeatureActive() && !$this->errors && $value) {
```

Comment:
```suggestion
        if (!Shop::isFeatureActive() && !$this->errors && $name) {
```
Related to my previous comment